docs/src/components/ColorSwatchFamily/ColorSwatchFamily.component.tsx

```diff
@@ -0,0 +1,87 @@
+// REACT
+import * as React from 'react';
+// VENDOR
+import styled, { css } from 'styled-components';
+import classNames from 'classnames';
+import { ColorContrastHelpers } from '../ColorContrastHelpers';
+import { ColorSwatch } from '../ColorSwatch';
+// THEME
+import { colors, sizes } from '../../../../src/theme';
+
+
+// INTERFACES
+interface ColorSwatchFamilyProps {
+  baseColorHexCode: string;
+  baseColorName: string;
+  className?: string;
+  copyChoice: string;
+  darkColorHexCode?: string;
+  lightColorHexCode?: string;
+}
+
+
+// STYLES
+const StyledColorSwatchFamily = styled.div`
+  padding: 12px;
+  margin: 24px;
+  width: 25%;
+  min-width: 300px;
+  display: inline-block;
+  box-shadow: 0px 0px 12px 4px ${colors.ash.light};
+`;
+
+
+// JSX MARKUP
+export const ColorSwatchFamily = ({ baseColorHexCode, baseColorName, className, copyChoice, darkColorHexCode, lightColorHexCode }: ColorSwatchFamilyProps): React.ReactElement<any> => (
+  <StyledColorSwatchFamily
+    baseColorName={baseColorName}
+    baseColorHexCode={baseColorHexCode}
+    className={classNames(className)}
+    darkColorHexCode={darkColorHexCode}
+    lightColorHexCode={lightColorHexCode}
+  >
+
+    <ColorSwatch
+      className={classNames(className)}
+      backgroundColor={baseColorHexCode}
+      contrastOnBlack={ColorContrastHelpers.getRoundedColorContrast(baseColorHexCode, '#000000')}
```

Comment:
I also think that the `contrastOnX` and `textColor` props should instead be calculated in the child `ColorSwatch`, rather than calculated here. All of them are just a function of the backgroundColor, so it wouldn't make sense to pass in any other values for a given color.

That way it's also much easier to change which colors we decide to contrast against in the future.

Reply:
+1